using MessageList

Hi,

I tried to use the MessageList Example today and couldn't bring it to work. I am using Windows 7 with the latest WebWorks 2.1.1.38.

What I did: 
- created the webworks.message.list 
- copied the library.xml in the new created folder
- created a webworks/ folder and copied the content of Smartphone\MessageList\src\webworks into it.

When I try to compile the example I am getting following error: 

:\Program Files\Research In Motion\BlackBerry WebWorks Packager>bbwp.exe
c:\Web
Works\messagelist\messagelist.zip
[INFO]                  Parsing command line options
[INFO]                  Parsing bbwp.properties
[INFO]                  Validating application archive
[INFO]                  Parsing config.xml
[INFO]                  Populating application source
[INFO]                  Compiling BlackBerry WebWorks application
C:\Users\znetbook\AppData\Local\Temp\widgetGen.10205088721315740929056.tmp\black
berry\web\widget\autogen\WidgetConfigAutoGen.java:33: package
webworks.message.l
ist does not exist
        _widgetExtensions.addElement(new
webworks.message.list.MessageListExtens
ion());
                                                              ^
1 error
Error!: Error: java compiler failed: C:\Program
Files\Java\jdk1.6.0_26\bin\javac
.exe @C:\Users\znetbook\AppData\Local\Temp\rapc_4b9c115e ...
[ERROR]                 RAPC exception occurred

C:\Program Files\Research In Motion\BlackBerry WebWorks Packager>

Any suggestions?  Do I miss something? 

Thanks.

bert2002

--- a/Smartphone/MessageList/README.md
+++ b/Smartphone/MessageList/README.md
@@ -32,7 +32,15 @@ Have a problem with this extension?  [Log an Issue](https://github.com/blackberr
 
 5. Copy the _**library.xml**_ file from the downloaded _**Smartphone\MessageList**_ directory to your new _**ext\webworks.message.list**_ directory
 
-6. Copy the downloaded _**Smartphone\MessageList\src\webworks**_ directory to your new _**ext\webworks.message.list\webworks**_ directory
+6. Copy the downloaded _**Smartphone\MessageList\src**_ directory to your new _**ext\webworks.message.list**_ directory
+
+Your Directory should look like this:
+
+* C:\Program Files\Research In Motion\BlackBerry WebWorks Packager\ext\webworks.message.list\
+
+	** src
+
+	** library.xml
 
 **NOTE:** Be sure to back-up this _**ext\webworks.message.list**_ directory in your WebWorks SDK extensions directory before performing a WebWorks SDK upgrade. Simply copy it back into the _**ext**_ directory after you have completed your SDK upgrade.
 

